pyprof/prof/pointwise.py

```diff
@@ -158,7 +158,8 @@ def __init__(self, d):
             _, t0, s0, dt0 = Pointwise.foo(in0)
             _, t1, s1, dt1 = Pointwise.foo(in1)
             assert (t0 == t1 == "tensor")
-            assert (dt0 == dt1)
+            # TODO: review assertions given that dtypes do not need to be equal
```

Comment:
TODOs should have a specified user story or bug in the comment

Reply:
@ceevee830 Not for an open source project. We shouldn't add company internal bug ID or tracking to public facing source code.

Reply:
However, don't submit commented code. Just remove it. Use the source repository to pull it back if need be.